### PR TITLE
[12.0] [MIG] sale_partner_incoterm

### PR DESCRIPTION
**Pull request by i-vyshnevska**
migrate to v12

### CODE REVIEW

File: sale_partner_incoterm/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```
Comment (nikul-serpentcs) [inline sale_partner_incoterm/__init__.py]: @i-vyshnevska no need to `coding` remove all py file

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

**Review by gurneyalex — APPROVED** [review]

File: sale_partner_incoterm/__init__.py (hunk 1 above)
Comment (i-vyshnevska) [reply sale_partner_incoterm/__init__.py]: thanks, updated

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@i-vyshnevska commit message should be `[12.0] [MIG] sale_partner_incoterm`.
Current one says nothing :wink:

**Comment by simahawk** [comment]
lint is red

**Comment by pedrobaeza** [comment]
Red lint can be due to https://github.com/OCA/maintainer-quality-tools/issues/589

About the commit message, I prefer standard Odoo format (https://www.odoo.com/documentation/12.0/reference/guidelines.html#commit-message-structure):

`[MIG] sale_partner_incoterm: Migration to 12.0`

**Comment by simahawk** [comment]
you've beaten me w/ the link to https://github.com/OCA/maintainer-quality-tools/issues/589 :smile: 

as for commit msg: fine too... the important thing is that the module name is there

**Comment by pedrobaeza** [comment]
MQT problem fixed

**Pull request merged**